examples/chat-room-reactions/javascript/src/script.ts

```diff
@@ -0,0 +1,54 @@
+import * as Ably from 'ably';
+import { ChatClient, RoomOptionsDefaults } from '@ably/chat';
+import { nanoid } from 'nanoid';
+import './styles.css';
+
+const realtimeClient = new Ably.Realtime({
+  clientId: nanoid(),
+  key: import.meta.env.VITE_PUBLIC_ABLY_KEY as string,
+});
+const chatClient = new ChatClient(realtimeClient);
+const room = chatClient.rooms.get('chat-room-reactions', RoomOptionsDefaults);
+
+/** 💡 Add every every room reaction published to the room 💡 */
+room.reactions.subscribe((reaction) => {
+  const reactionsContainer = document.getElementById('reaction-area');
+
+  const reactionElement = document.createElement('span');
+  reactionElement.className = 'reaction';
+  reactionElement.textContent = reaction.type;
+  reactionElement.dataset.createdAt = new Date().toISOString();
+  reactionsContainer.appendChild(reactionElement);
+});
+
+/** 💡 Attach to the room to subscribe to reactions 💡 */
+await room.attach();
+
+setInterval(() => {
```

Comment:
What about having this inside the subscribe block? Would need a small rework, but might make things a bit simpler? You can then just use a setTimeout, what do you think? :)

Reply:
Updated!